moveit_ros/benchmarks/launch/run_benchmark.launch.py

```diff
@@ -0,0 +1,94 @@
+import os
```

Comment:
as discussed, this should probably be moved into moveit_resources along with the db file.

Reply:
shouldn't this file be removed if it's part of resources or the tutorial?